install/collect_mac_addresses_for_ncns.md

```diff
@@ -130,11 +130,11 @@ making a backup of them, in case they need to be examined at a later time.
 
 1. Check that IP addresses are set for each interface and investigate any failures.
 
-  > Note that bond0.can0 is optional in CSM 1.2+
+    > **`Note:`** Note that bond0.can0 is optional in CSM 1.2+
```

Comment:
It seems redundant to use note twice here. Also, a period is needed at the end of the line. 

Suggested change:

> **`Note:`** bond0.can0 is optional in CSM 1.2+ releases.

Reply:
Corrected